admin/BundleSearchIndexesDump

```diff
@@ -187,7 +193,7 @@ for my $hash_program ('md5sum', 'sha256sum') {
     my $hash_output_file = uc($hash_program . 's');
     chomp (my $hash_bin = `which g$hash_program` || `which $hash_program`);
     my $hash_command = "cd $working_dir" .
-        " && $hash_bin --binary *.tar.zst > $hash_output_file";
+        " && ionice -c3 $hash_bin --binary *.tar.zst > $hash_output_file";
```

Comment:
`ioniceit` is missing here.
It would be more convenient to have it as a small executable script instead of a function, to avoid duplication.

Reply:
Added missing call and moved definition to a file in commit https://github.com/metabrainz/musicbrainz-server/pull/2350/commits/7da6795190217c968ae83855bbe6bf08068a8681.